megatron/global_vars.py

```diff
@@ -244,7 +244,7 @@ def write(self, names, writer, iteration, normalizer=1.0, reset=False):
         assert normalizer > 0.0
         for name in names:
             value = self.timers[name].elapsed(reset=reset) / normalizer
-            writer.add_scalar(name + '-time', value, iteration)
+            writer.add_scalar('time/' + name + '-time', value, iteration)
```

Comment:
```suggestion
            writer.add_scalar(f'time/{name}-time', value, iteration)
```

if we are adding `(s)` in other time logs, should there be one here? Or nowhere and we know it's always secs?
The other place you added it is in `iteration-time/iteration-time (s)`

Also suggesting a consistent use of f-strings.

Reply:
yes, as far as i understand, all time-related metrics are reported in seconds, so removing these `(s)`